### ISSUE TITLE
Draft onefact.org Landing Page

### ISSUE THREAD

**Issue by jaanaltosaar** [onefact/onefact.org#7]
focused, capable, partners

ways we talk about one fact foundation:

https://docs.google.com/document/d/1Ubgrb8DtueoFSfc4cbP9gebTkdUZgOAp48pkYt0rjWE/edit?usp=sharing

# deck 
deck.onefact.org -- svelte.js

# initiatives 
* payless health [ talk ] 
* data thinking [ blurb from class ] -- open access class, teaching 150+ MBA students to PhD students to faculty members in 2023 at top international institutions and learners joining from all over the world.
* clinicalbert -- support from the researchallofus.org principle investigator to train on all of us

# funding & partners 
* dolthub 
* yale medicine 
* NIH AIM-AHEAD - https://m.yale.edu/bky5 - uche talk
* five boro bike tour

# programs we support
* the late pass
* arbiter.report

# certifications & registrations
* add EIN and add 501c3 status and https://www.irs.gov/pub/irs-soi/eo_de.csv

**Comment by jaanaltosaar** [comment]
Based on these learner definitions, here is an operational definition of "data thinking" for our class:

Data thinking is the mental process of analyzing, interpreting, and deriving meaning from data in order to understand and make decisions about the world. It involves:

Converting data into information and knowledge through logical and sometimes creative thinking. This includes categorizing, computing, reasoning, rationalizing, as well as making correlations and intuiting causation.

Asking questions about how the data was collected and making assumptions in order to comprehend its meaning. This includes considering what is included or excluded and why.

-Connecting datasets and individual data points together through grouping, categorizing, and storytelling in order build a coherent understanding.

-Translating data insights into subjective experiences and using them to explain, predict, gain perspective, or make judgements about real-world phenomenon.

Recognizing that data and thinking are interconnected and co-constructed. Data both shapes and is shaped by how we think about and make sense of the world.

Continually optimizing and reconsidering one's mental models in light of new data. Data thinking is an iterative process of reframing one's understanding.

In summary, data thinking is the active, inquisitive, and meaning-making cognitive process of understanding the world through data. It is a skill that can be developed and honed over time with practice.
